docs/api/v3.rst

```diff
@@ -1814,7 +1814,7 @@ Remote Repository listing
     :query string vcs_provider: return remote repositories for specific vcs provider (``github``, ``gitlab`` or ``bitbucket``)
     :query string organization: return remote repositories for specific remote organization (using remote organization ``slug``)
     :query string expand: allows to add/expand some extra fields in the response.
-                          Allowed values are ``project`` and ``organization``.
+                          Allowed values are ``projects`` and ``remote_organization``.
                           Multiple fields can be passed separated by commas.
```

Comment:
@agjohnson this a change in the API request/response required to make sense with the new relation. It also reserves `?expand=organization` to be used for `Organization` objects.

Reply:
I was going to say we should keep `project` around for backwards compatibility, but this API should really only be internal use anyways. So yeah, agreed this seems fine.